src/digitalmarketplace/components/alert/template.njk

```diff
@@ -0,0 +1,10 @@
+<div data-module="dm-alert" class="dm-alert dm-alert--{{ params.type }}{%- if params.classes %} {{ params.classes }}{% endif %}"{%- for key, value in params.attributes %} {{key}}="{{value}}"{% endfor %}>
+    <h{% if params.headingLevel %}{{ params.headingLevel }}{% else %}2{% endif %} class="dm-alert--title">
```

Comment:
I think the `dm-alert--title` should be `dm-alert__title`. The title is an element of component not a modification of the component.